src/repl/repl_commands.cpp

```
@@ -1023,6 +1068,25 @@ bool dispatch_command(const std::string& command, const std::string& arg, ReplSt
+  } else if (command == "spinner") {
+    LOG_FEATURE("cmd_spinner");
+    if (arg.empty()) {
+      std::string current = s.spinner_name.empty() ? (s.bofh ? "bofh" : "default") : s.spinner_name;
+      s.out << "[spinner: " << current << "]\n";
+      s.out << "Usage: /spinner <name> — loads .config/spinners/<name>.txt\n";
+      s.out << "Built-in: default, bofh. Drop .txt files for custom personalities.\n";
+    } else if (arg == "default" || arg == "off") {
+      s.spinner_name.clear();
+      s.bofh = false;
+      s.out << "[spinner: default]\n";
+    } else {
```

Clang (14.0.6)

[note] +1, nesting level increased to 2

(clang)